docs/reference/search/rrf.asciidoc

```diff
@@ -1,6 +1,423 @@
+                {
+                    "key" : 1,
+                    "doc_count" : 3
+                },
+                {
+                    "key" : 2,
+                    "doc_count" : 2
+                }
+            ]
+        }
+    }
+}
+----
+// TESTRESPONSE[s/: \.\.\./: $body.$_path/]
+
+Let's break down how these hits were ranked. We
+start by running the standard query and knn search
+separately to collect what their individual hits are.
+
+First, we look at the hits for the standard query.
+
+[source,console-result]
+----
+"hits" : [
+    {
+        "_index" : "rrf-index",
+        "_id" : "4",
+        "_score" : 0.16152832,              <1>
+        "_source" : {
+            "rrf_integer" : 2,
+            "rrf_text" : "rrf rrf rrf rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "3",                        <2>
+        "_score" : 0.15876243,
+        "_source" : {
+            "rrf_integer" : 1,
+            "rrf_vector" : [3],
+            "rrf_text" : "rrf rrf rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "2",                        <3>
+        "_score" : 0.15350538,
+        "_source" : {
+            "rrf_integer" : 2,
+            "rrf_vector" : [4],
+            "rrf_text" : "rrf rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "1",                        <4>
+        "_score" : 0.13963442,
+        "_source" : {
+            "rrf_integer" : 1,
+            "rrf_vector" : [5],
+            "rrf_text" : "rrf"
+        }
+    }
+]
+----
+// TEST[skip:example fragment]
+<1> rank 1, `_id` 4
+<2> rank 2, `_id` 3
+<3> rank 3, `_id` 2
+<4> rank 4, `_id` 1
+
+Note that our first hit doesn't have a value for the `rrf_vector` field. Now,
+we look at the results for the knn search.
+
+[source,console-result]
+----
+"hits" : [
+    {
+        "_index" : "rrf-index",
+        "_id" : "3",                   <1>
+        "_score" : 1.0,
+        "_source" : {
+            "rrf_integer" : 1,
+            "rrf_vector" : [3],
+            "rrf_text" : "rrf rrf rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "2",                   <2>
+        "_score" : 0.5,
+        "_source" : {
+            "rrf_integer" : 2,
+            "rrf_vector" : [4],
+            "rrf_text" : "rrf rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "1",                   <3>
+        "_score" : 0.2,
+        "_source" : {
+            "rrf_integer" : 1,
+            "rrf_vector" : [5],
+            "rrf_text" : "rrf"
+        }
+    },
+    {
+        "_index" : "rrf-index",
+        "_id" : "5",                   <4>
+        "_score" : 0.2,
+        "_source" : {
+            "rrf_integer" : 1,
+            "rrf_vector" : [1]
+        }
+    }
+]
+----
+// TEST[skip:example fragment]
+<1> rank 1, `_id` 3
+<2> rank 2, `_id` 2
+<3> rank 3, `_id` 1 (tiebreaker where `1` is less than `5` for `_id`)
```

Comment:
I think we tiebreak by  shard id and Lucene doc ID, not by ES `_id` field, but it may happened so that here smaller ids also corresponded to smaller Lucene doc ids.

Reply:
Thank you for catching this! I changed the example slightly so there would be no tiebreakers required as it doesn't really add anything to the example anyway.